packages/internals/support/src/entities/component.js

```diff
@@ -61,11 +61,17 @@ class Component extends Entity {
   }
 
   getViews() {
-    // TODO
+    let views = new FileCollection();
+    const viewConfig = this.getConfig().views;
```

Comment:
just wondering.... should this get the `views` prop directly from the config or should it use `this.get('views.filter')`? I.e. if people override this or define a getter should that be the value that is used as the filter?